src/Ztm.WebApi/AddressPools/ReceivingAddress.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+using NBitcoin;
+
+namespace Ztm.WebApi.AddressPools
+{
+    public sealed class ReceivingAddress
+    {
+        public Guid Id { get; }
+        public BitcoinAddress Address { get; }
+        public bool IsLocked { get; }
+        public List<ReceivingAddressReservation> ReceivingAddressReservations { get; }
```

Comment:
Better to use `IEnumerable<ReceivingAddressReservation>` if mutable is not required; otherwise better to use `ICollection<ReceivingAddressReservation>` instead.